samples/sensor/lm73c/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: lm73c
+tests:
+  sample.sensor.lm73c:
+    harness: console
+    tags: sensors
+    harness_config:
+        type: one_line
+        regex:
+            - "ttemp: (.*)"
+        fixture: fixture_i2c_lm73c
+    depends_on: i2c
+    filter: dt_compat_enabled("ti,lm73c")
```

Comment:
I don't think this will pick up the boards you added overlays for. Please replace this with a platform_allow list

Reply:
This is true; only certain boards are normally tested.  But I believe we're supposed to be avoiding platform_allow lists.

If you make it `frdm_k64f` instead of (or in addition to) `frdm_kw41z` then at least one will be tested.  

I don't know where the list of tested reference boards is hidden.

Reply:
Should I just delete the sample.yaml then? use the one from the 9808

sample:
  name: MCP9808 Temperature Sensor
tests:
  sample.sensor.mcp9808:
    harness: sensor
    tags: sensors
    depends_on: i2c
    filter: dt_compat_enabled("microchip,mcp9808")


I was looking at bme280 and mcp9808, thought I was taking the best of both worlds

Reply:
No, the sample.yaml needs to be there.  It just needs to have at least one board for which there's a devicetree node, so that this command does something:

```
tirzah[882]$ twister -T samples/sensor/lm73c/
Renaming output directory to /mnt/nordic/zp/zephyr/twister-out.12
INFO    - Zephyr version: zephyr-v2.4.0-2711-g8f057fcdea
INFO    - JOBS: 32
INFO    - Selecting default platforms per test case
INFO    - Building initial testcase list...
INFO    - 1 test suites (24 configurations) selected, 23 configurations discarded due to filters.
INFO    - Adding tasks to the queue...
INFO    - Added initial list of jobs to queue
INFO    - Total complete:    1/   1  100%  skipped:   24, failed:    0
INFO    - 0 of 0 test configurations passed (0.00%), 0 failed, 24 skipped with 0 warnings in 2.03 seconds
INFO    - In total 0 test cases were executed, 24 skipped on 24 out of total 318 platforms (7.55%)
INFO    - 0 test configurations executed on platforms, 0 test configurations were only built.
INFO    - Saving reports...
INFO    - Writing xunit report /mnt/nordic/zp/zephyr/twister-out/twister.xml...
INFO    - Writing xunit report /mnt/nordic/zp/zephyr/twister-out/twister_report.xml...
INFO    - Writing JSON report /mnt/nordic/zp/zephyr/twister-out/twister.json
INFO    - Run completed
```